erizo/src/erizo/rtp/LayerDetectorHandler.cpp

```diff
@@ -34,16 +34,11 @@ void LayerDetectorHandler::disable() {
 void LayerDetectorHandler::read(Context *ctx, std::shared_ptr<DataPacket> packet) {
   RtcpHeader *chead = reinterpret_cast<RtcpHeader*>(packet->data);
   if (!chead->isRtcp() && enabled_ && packet->type == VIDEO_PACKET) {
-    RtpHeader *rtp_header = reinterpret_cast<RtpHeader*>(packet->data);
-    RtpMap *codec = stream_->getRemoteSdpInfo()->getCodecByExternalPayloadType(rtp_header->getPayloadType());
-    if (codec && codec->encoding_name == "VP8") {
-      packet->codec = "VP8";
+    if (packet->codec = "VP8") {
```

Comment:
`==` ?

Reply:
yeah, I'm trying to do my best in recreating the prs from my own deviated source code. fixed typos